spec/features/friends_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+RSpec.describe 'Friends' do
+  let(:user) { create(:user) }
+
+  describe 'dashboard' do
+    it 'Should default to no friends' do
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+      visit '/dashboard'
+
+      expect(page).to have_content('Your Friends:')
+      expect(page).to have_content('You currently have no friends.')
+      expect(page).to have_field('friend_email')
+      expect(page).to have_button('Add Friend')
```

Comment:
this test, and subsequent tests might be slightly better if we used a 'within "#friends" do' clause for this and subsequent tests.  this is very low priority though, and it might not be worth the effort.